test/test_google_drive.rb

```diff
@@ -394,7 +397,7 @@ def get_session
         )
       end
 
-      @@session = GoogleDrive::Session.from_config(config_path)
+      @@session = GoogleDrive::Session.from_config('config.json')
```

Comment:
Why did you change here?

Reply:
The error message indicates that **test**/config.json is invalid as JSON. Maybe you put a broken file there? Note that you must put the config file inside "test" directory.

Reply:
Ah whoops...missed the part about the "test" directory.  Thanks for the help -- I've restored the `config_path` variable now.